Lib/test/test_copy_fallback.py

```diff
@@ -0,0 +1,60 @@
+"""Unit tests for the copy module without the C accelerator."""
```

Comment:
We use a different approach to test pure Python and C accelerated code:

* create a test class for C and Python implementation
* remove ``import copy``
* replace ``copy.*`` with ``self.module....``

```
py_copy = import_helper.import_fresh_module(
    'copy', blocked=['_copy']
)
c_copy = import_helper.import_fresh_module(
    'copy', fresh=['_copy']
)


class TestCopy:
    module = None

    def test_exceptions(self):
        self.assertIs(self.module.Error, self.module.error)
        self.assertTrue(issubclass(self.module.Error, Exception))


class TestCopyPy(TestCopy, unittest.TestCase):
    module = py_copy


@unittest.skipUnless(c_copy, 'requires _copy')
class TestCopyC(TestCopy, unittest.TestCase):
    module = c_copy
```